packages/timeline-state-resolver/src/integrations/httpWatcher/index.ts

```diff
@@ -26,38 +33,8 @@ export class HTTPWatcherDevice extends Device<DeviceOptionsHTTPWatcherInternal>
 	private interval: NodeJS.Timer | undefined
 	private status: StatusCode = StatusCode.UNKNOWN
 	private statusReason: string | undefined
-	constructor(
-		deviceId: string,
-		deviceOptions: DeviceOptionsHTTPWatcherInternal,
-		getCurrentTime: () => Promise<number>
-	) {
-		super(deviceId, deviceOptions, getCurrentTime)
-		const opts = deviceOptions.options
-		switch (opts?.httpMethod) {
-			case 'post':
-				this.httpMethod = TimelineContentTypeHTTP.POST
-				break
-			case 'delete':
-				this.httpMethod = TimelineContentTypeHTTP.DELETE
-				break
-			case 'put':
-				this.httpMethod = TimelineContentTypeHTTP.PUT
-				break
-			case 'get':
-			case undefined:
-			default:
-				this.httpMethod = TimelineContentTypeHTTP.GET
-				break
-		}
-
-		this.expectedHttpResponse = Number(opts?.expectedHttpResponse) || undefined
-		this.headers = opts?.headers
-		this.keyword = opts?.keyword
-		this.intervalTime = Math.max(Number(opts?.interval) || 1000, 1000)
-		this.uri = opts?.uri
-	}
 
-	onInterval() {
+	#onInterval() {
```

Comment:
I am personally not a fan of using these private modifiers because they are so inconsistent with typescript. I wish typescript would still let you type them as private explicitly because I get a stroke reading this. Not a real issue with using them here, I just wish the world was a less triggering place.

Reply:
I've changed them back. I do quite like them, but as I don't do much in this repository its better to stick to the trend of the other devices.